### DISCUSSION TITLE
Input normalization

### DISCUSSION THREAD

**Discussion by MaximeLee** [lululxvi/deepxde#1189] [Ideas]
Hi,

I was wondering. Is there some kind of normalization procedure in Deepxde?
And would it be a good idea to add BatchNormalization layers in the Neural networks?

Best regards

**Comment by lululxvi** [comment]
DeepXDE supports batch norm, but I don't think it is a good idea to use batch norm in PINNs.

**Reply by dang-trinh156** [reply]
Can you please elaborate why it's not a good idea?

**Reply by lululxvi** [reply]
Simply based on my past limited experience.

**Reply by dang-trinh156** [reply]
Thank you, sir!
